src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -1,198 +1,277 @@
-package ca.team2706.frc.robot.config;

```

Comment:
I think your merge went wrong here because all the changes from master got deleted